### ISSUE TITLE
[ERROR] Unauthenticated

### ISSUE THREAD

**Issue by NullHF** [netindev/drill#3]
I very often get this error :

C:\Users\Gaming\Downloads>java -jar drill-0.0.2-SNAPSHOT.jar -host xmr.crypto-pool.fr -port 3333 -user 49tHfACjLUFhZGNGGaVKC1JEUGonvmUpJSYi63vuh1bt4YEnKrwkP9wHzEB7HdX3CXbLp7tiitmcjToPHr4Dphm63TfsKBV -pass x
Drill cryptonight miner, written by netindev, V: 0.02
[22:24:13] [INFO] Connected to: xmr.crypto-pool.fr:3333
[22:24:13] [INFO] New job received, diff: 18000
[22:24:28] [INFO] Result accepted by the pool!
[22:25:45] [INFO] Result accepted by the pool!
[22:27:12] [INFO] New job received, diff: 18000
[22:27:58] [ERROR] Unauthenticated
[22:32:46] [INFO] New job received, diff: 18000
[22:32:47] [INFO] New job received, diff: 18000
[22:32:52] [ERROR] Unauthenticated
[22:33:28] [INFO] New job received, diff: 18000
[22:34:06] [INFO] New job received, diff: 18000
[22:34:06] [INFO] New job received, diff: 18000
[22:35:11] [ERROR] Unauthenticated
[22:36:29] [ERROR] Unauthenticated
[22:36:57] [INFO] New job received, diff: 18000
[22:38:10] [ERROR] Unauthenticated
[22:38:44] [INFO] New job received, diff: 18000
[22:38:53] [ERROR] Unauthenticated
[22:40:20] [INFO] New job received, diff: 18000
[22:40:49] [INFO] New job received, diff: 18000
[22:41:01] [ERROR] Unauthenticated
[22:42:34] [ERROR] Unauthenticated
[22:42:53] [ERROR] Unauthenticated
[22:44:47] [INFO] New job received, diff: 18000
[22:45:50] [ERROR] Unauthenticated
[22:45:55] [INFO] New job received, diff: 18000

I compiled the src code myself for test but is same

**Comment by netindev** [comment]
very strange.. the same error for me with your pool.
i tried with moneroocean and works pretty well (no Unauthenticated error).
soon i fix this.

**Comment by NullHF** [comment]
> very strange.. the same error for me with your pool.
> i tried with moneroocean and works pretty well (no Unauthenticated error).
> soon i fix this.

So you think the pool i use is not very good ?

**Comment by netindev** [comment]
no. the problem is the miner, try using another miner or use another pool.

**Comment by NullHF** [comment]
Just a question , is CPU miner right ? you can implemante gpu ?

**Comment by netindev** [comment]
soon, yeah i can implement it.
